exampleCourse/questions/element/graph/server.py

```diff
@@ -42,7 +42,7 @@ def generate(data):
     data["params"]["multigraph"] = pl.to_json(multigraph)
 
     # Generation code for color graph
-    subset_sizes = [5, 5, 4, 3, 2, 4, 4, 3]
+    subset_sizes = (5, 5, 4, 3, 2, 4, 4, 3)
```

Comment:
I'm curious, why this change, and why are this and `subset_color` different? I reverted this change locally and didn't see any lint errors.

Reply:
To be honest, I am not sure 😅